lib/demos/todo.dart

```diff
@@ -12,23 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import 'dart:html' as html;
-
 import 'package:butterfly/butterfly.dart';
-import 'store.dart';
 
 // TODO(yjbanov): maket these injectable
 Store store = new Store();
```

Comment:
Hmm. Had to roll back. Analyzer fails to infer the type, even when I make it `final`.